addons/mail/static/src/new/discuss/discuss.js

```diff
@@ -74,6 +74,7 @@ export class Discuss extends Component {
         useEffect(
             () => {
                 if (
+                    this.store.discuss.setDoneInDiscuss &&
```

Comment:
I believe this condition would be better done as checking the current discuss thread is inbox, rather than adding an extra field.

Also the extra field might fail, because `setDone` could be called when the counter is not yet 0, and will stay at `true` even later, when maybe the current thread is no longer inbox.